### ISSUE TITLE
[Effect Suggestion] Bouncy screen

### ISSUE THREAD

**Issue by KatieDeForest** [gta-chaos-mod/ChaosModV#2523]
**Name of effect**
Potential name(s) for the effect
Shaky shaky
Bouncy screen
Shaking screen
Shaking around
Bouncing around
Shaking camera
Bouncing camera

**Duration (choose one and replace the corresponding "[ ]" with "[X]")**
- [X] Long
- [ ] Short
- [ ] No duration

**Effect Description**
Description of the effect; what should it do?

Make the player screen a tiny bit smaller, and then make it bounce around. So it's still possible to see, but super disturbing when driving or aiming. 
Basically saying it's a smaller resolution bouncing around in a bigger resolution. The empty space would just be pure black background, but the actual game screen will bounce around, giving this half bouncy half shaking feeling.
Hope this is possible, would be hilarious. Don't make the bounce area too big tho, then it's just a giant DVD screen. It would only work if they are almost the same resolution.

**Comment by Last0xygen** [comment]
This is currently not possible
